tests/test_record_requests_for_all_methods.py

```diff
@@ -45,6 +45,48 @@ def test_write(self, cassette: Cassette):
         Path(cassette.storage_file).unlink()
 
 
+@record_requests_for_all_methods()
```

Comment:
please add there one more decorator, to ensure that values are not 2

Reply:
I've added another class for that.